src/open_anything_diffusion/models/flow_trajectory_predictor.py

```diff
@@ -243,3 +242,26 @@ def predict_step(self, xyz: torch.Tensor, mask: torch.Tensor) -> torch.Tensor:
         with torch.no_grad():
             flow = self.forward(batch)
         return flow
+
+
+class FlowSimulationInferenceModule(L.LightningModule):
```

Comment:
Why not just use the above "predict_step" method?

Reply:
Hmmm, cuz other simulator that generates groundtruth will need more information than what the prediction process needs, and therefore using this is because I want to unify the simulators' api